src/cpp/src/continuous_batching/model_runner.hpp

```diff
@@ -254,6 +339,64 @@ class ModelRunner {
 
                 output_seq_len = 0;
                 Sequence::CPtr sequence = running_sequences[seq_idx];
+                if (m_is_hidden_state_export_needed) {
+                    size_t start_token_idx = current_token_idx;
+                    size_t sequence_length = num_scheduled_tokens;
+
+                    auto key = std::make_pair(sequence_group->get_request_id(), sequence->get_grouped_id());
+                    m_sequence_hidden_state_mapping[key] = std::make_pair(start_token_idx, sequence_length);
+                }
+                if (m_is_hidden_state_import_needed && hidden_state_data && hidden_size > 0) {
```

Comment:
Could you please add a description to if statements, or put `m_is_hidden_state_import_needed && hidden_state_data && hidden_size > 0` into a descriptive variable/function like `should_import_hidden_state`, `get_stored_hidden_state`, or `is_valid_hidden_state`